kpops/components/streams_bootstrap/streams/streams_app.py

```diff
@@ -152,3 +156,19 @@ async def clean(self, dry_run: bool) -> None:
         """Destroy and clean."""
         await super().clean(dry_run)
         await self._cleaner.clean(dry_run)
+
+    @override
+    def manifest_deploy(self) -> Resource:
+        manifest = super().manifest_deploy()
+        operation_mode = get_config().operation_mode
+
+        if operation_mode is OperationMode.ARGO:
```

Comment:
we don't need to check the operation mode here and again inside the manifestor, one check should be sufficient. But now that I see it I'll take my proposal back, I think the additional abstraction doesn't make that much sense as I thought. we can also discuss it more next week :)